mrc/control/acting/strategies.py

```diff
@@ -0,0 +1,46 @@
+from mrc.control.acting.abstract import AbstractStrategy
+from mrc.control.movement_prediction.target_position_calculator import TargetPositionCalculator
+from mrc.control.steering.exceptions import ObstacleOnTheWayException, SteeringException
+from mrc.shared.position import PolarPosition
+
+
+class FollowMasterStrategy(AbstractStrategy):
+
+    def __init__(self, steering_strategy, locator, configurator):
+        self.steering_strategy = steering_strategy
+        self.locator = locator
+        self.configurator = configurator
+        self.position_calculator = TargetPositionCalculator(self.configurator)
+        self.locations = None
```

Comment:
`self.locations` can be converted to local variable

Reply:
Or no, it can be actually removed completely, along with its usage in `read(...)`

Reply:
It shall be used but i have forgotten about it, thanks!